airflow-core/docs/core-concepts/auth-manager/index.rst

```diff
@@ -170,8 +170,76 @@ cookie named ``_token`` before redirecting to the Airflow UI. The Airflow UI wil
     return response
 
 .. note::
-    Do not set the cookie parameter ``httponly`` to ``True``. Airflow UI needs to access the JWT token from the cookie.
+  Ensure that the cookie parameter ``httponly`` is set to ``True``. UI no longer manages the token.
```

Comment:
I think this also needs to change. I initially considered a separate PR without backporting, but since this is after 3.1.1 and v-3-1-test is already past 3.1.4, it should be fine.